### ISSUE TITLE
issue importing scikit-image

### ISSUE THREAD

**Issue by segalinc** [scikit-image/scikit-image#5533]
Hello, 

I am trying to use a package called `face_alignment` which uses scikit-image. 
Until few months ago it was working fine. Today I installed it in a new virtual environment and I get this error using all versions >= 0.17.2
Not sure what' happening here.
Can you help?

```
import face_alignment
lib/python3.8/site-packages/face_alignment/__init__.py:7: in <module>
    from .api import FaceAlignment, LandmarksType, NetworkSize
lib/python3.8/site-packages/face_alignment/api.py:4: in <module>
    from skimage import io
lib/python3.8/site-packages/skimage/__init__.py:135: in <module>
    from .data import data_dir
lib/python3.8/site-packages/skimage/data/__init__.py:64: in <module>
    from pooch.utils import file_hash
E   ImportError: cannot import name 'file_hash' from 'pooch.utils' (/lib/python3.8/site-packages/pooch/utils.py)
```

**Comment by hmaarrfk** [comment]
I'm really sorry you are hitting this.

We are trying to address this issue as we speak.

https://github.com/scikit-image/scikit-image/pull/5531

But for yourself, if you could install `pooch!=1.5.0` that would work.

Otherwise, you could edit the file:
```
lib/python3.8/site-packages/skimage/data/__init__.py
```
and change
```
from pooch.utils import file_hash
```
to
```
from pooch import file_hash
```

**Comment by segalinc** [comment]
yes I am trying to see if changing the pooch version before installing the other package can work.
Thank you for the prompt reply  for now

**Comment by cheng-chi** [comment]
Installing pooch==1.4.0 works for me

**Comment by segalinc** [comment]
Yeah it's just not super nice to have to install not related packages to make one work.. for now it will do...

**Comment by Klinggaard** [comment]
So I checked the pooch source code and the file_hash functionality has been moved from utils.py to hashes.py.

This means that the import in scikit-image should be `from pooch import file_hash` instead of `from pooch.utils import file_hash`

**Comment by grlee77** [comment]
Thanks, @Klinggaard. We merged a fix as you suggest yesterday. We should have a v0.18.3 release out today (wheels are still building) with a fix. 

In the meantime, Pooch just released 1.5.1 which restores `pooch.utils.file_hash` with a deprecation warning. So updating to Pooch v1.5.1 should also fix the issue.

**Comment by hmaarrfk** [comment]
Because i know I've looked for this website alot, here is the build tracker, seems we are almost there with a few linux builds straggling behind

https://github.com/scikit-image/scikit-image/runs/3411192484

**Comment by hmaarrfk** [comment]
As always, release processes change and it seems that we aren't able to upload anymore:
https://github.com/scikit-image/scikit-image/runs/3408063253?check_suite_focus=true

We'll need to likely upload manually, or think of something different.

**Comment by grlee77** [comment]
I uploaded everything manually for PyPI just now. I assume conda-forge will get an autotick PR soon.

> As always, release processes change and it seems that we aren't able to upload anymore:
https://github.com/scikit-image/scikit-image/runs/3408063253?check_suite_focus=true

That was a prior build where i tried to backport the automated wheel building. For some reason the Python 3.9 wheels were skipped, though, so I reverted back to the prior script. The failure to upload was likely because we need to add some `twine`-related variables under the GitHub secrets for this repo.
